core/src/main/java/uk/ac/bbsrc/tgac/miso/core/util/EnaSubmissionPreparation.java

```diff
@@ -45,7 +45,7 @@
 import uk.ac.bbsrc.tgac.miso.core.data.type.HealthType;
 import uk.ac.bbsrc.tgac.miso.core.data.type.SubmissionActionType;
 
-public class EraSubmissionPreparation {
+public class EnaSubmissionPreparation {
   private abstract class ChildSubmissionFile {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Convert the abstract class "ChildSubmissionFile" into an interface. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1610)